src/OmniSharp.Dnx/DnxPaths.cs

```diff
@@ -138,31 +132,24 @@ private IEnumerable<string> GetRuntimeLocations()
             {
                 // Newer path
                 yield return Path.Combine(home, ".dnx");
+            }
 
-                // New path
-                yield return Path.Combine(home, ".k");
-
-                // Old path
-                yield return Path.Combine(home, ".kre");
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ProgramFiles")))
```

Comment:
The windows installer puts them in that location, but dnvm doesn't know about that location. Not sure if you should have all 3 or what.